bin/v-add-letsencrypt-domain

```diff
@@ -580,10 +580,10 @@ if [ "$?" -ne '0' ]; then
 fi
 
 # Adding LE autorenew cronjob
-if [ -z "$(grep v-update-letsen "/etc/cron.d/hestiaweb")" ]; then
+if [ -z "$(grep /usr/local/hestia/bin/v-update-letsencrypt "/etc/cron.d/hestiaweb")" ]; then
```

Comment:
Shouldn't we use $BIN here? Or is it not available? Eg:

https://github.com/hestiacp/hestiacp/blob/3370d3ddf6771fab091c2500622ffc13993d5519/bin/v-add-user#L190

Reply:
It runs directly as source so $BIN is not available yet.. How ever the if statement should be changed